### PR TITLE
Update mount parameters

### PR DESCRIPTION
**Pull request by TunaLobster**
A double build on the server will be required to remake the inventory.

### CODE REVIEW

**Comment by TunaLobster** [comment]
Changed centidegress to degrees where I caught it.

File: common/source/docs/common-camera-gimbal.rst [hunk 1]
```diff
@@ -100,12 +100,12 @@ If you find your gimbal is moving in the wrong direction, check the
 .. figure:: ../../../images/MPCameraAndGimbalSetupScreen.jpg
    :target: ../_images/MPCameraAndGimbalSetupScreen.jpg
 
-   Mission Planner: Camera andGimbal Setup Screen
+   Mission Planner: Camera and Gimbal Setup Screen
 
-If you wish to adjust the gimbal tilt, roll or pan while flying, you can
-set the :ref:`MNT_RC_IN_PAN<MNT_RC_IN_PAN>`, :ref:`MNT_RC_IN_ROLL<MNT_RC_IN_ROLL>`, and/or :ref:`MNT_RC_IN_TILT<MNT_RC_IN_TILT>` to the RC channel to be used for this. Be sure that those channels do not have any ``RCx_OPTION`` also assigned.
+If you wish to adjust the gimbal pan, tilt, or roll while flying, you can
+set the RC controls for pan, tilt, or roll using :ref:`RCx_OPTION<RC1_OPTION>`.
```
Comment (Hwurzburg) [inline common/source/docs/common-camera-gimbal.rst:106]:
```suggestion
set the RC controls for roll, tilt, or pan using ``RCx_OPTION`` 212 (Mount1 Roll), 213 (Mount1 Pitch), 214 (Mount1 Yaw) for the first mount, or 215-217 for the second mount, respectively.
```

File: common/source/docs/common-camera-gimbal.rst [hunk 2]
```diff
@@ -181,21 +181,21 @@ The camera/gimbal direction can be controlled by the pilot using RC control(RC T
 
 If a retractable mount is employed, the overall mount may be deployed or retracted using an output assigned with ``SERVOx_FUNCTION`` set to "MountOpen". This will be automatically controlled by the autopilot as if it were landing gear (see :ref:`common-landing-gear`), or by pilot using an rc channel whose ``RCx_OPTION`` is set to "Landing Gear".
 
-The default targeting mode for the camera/gimbal is set by the :ref:`MNT_DEFLT_MODE<MNT_DEFLT_MODE>` parameter.
+The default targeting mode for the camera/gimbal is set by the :ref:`MNT1_DEFLT_MODE<MNT1_DEFLT_MODE>` parameter.
```
Comment (Hwurzburg) [inline common/source/docs/common-camera-gimbal.rst:184]:
```suggestion
The default targeting mode for the first camera/gimbal is set by the :ref:`MNT1_DEFLT_MODE<MNT1_DEFLT_MODE>` parameter, while ref:`MNT2_DEFLT_MODE<MNT2_DEFLT_MODE>` is used for the second mount, if used.
```

File: common/source/docs/common-camera-gimbal.rst [hunk 3]
```diff
@@ -181,21 +181,21 @@ The camera/gimbal direction can be controlled by the pilot using RC control(RC T
...
 
 The direction the axes are set for the NEUTRAL and RETRACTED modes are set by:
```
Comment (Hwurzburg) [inline common/source/docs/common-camera-gimbal.rst:186]:
```suggestion
The direction the axes are set for the NEUTRAL and RETRACTED modes are set by (shown for the first mount):
```

File: common/source/docs/common-gremsy-pixyu-gimbal.rst [hunk 4]
```diff
@@ -25,26 +25,22 @@ We recommend connecting the Gimbals's COM2 port to one of the autopilot's Serial
 
 Connect with a ground station and set the following parameters:
```
Comment (Hwurzburg) [inline common/source/docs/common-gremsy-pixyu-gimbal.rst:26]:
```suggestion
Connect with a ground station and set the following parameters, if using the first mount:
```

File: common/source/docs/common-gremsy-pixyu-gimbal.rst [hunk 5]
```diff
@@ -25,26 +25,22 @@ We recommend connecting the Gimbals's COM2 port to one of the autopilot's Serial
...
 
-- :ref:`MNT_TYPE <MNT_TYPE>` to "4" for "SToRM32 MavLink" and reboot the autopilot
+- :ref:`MNT1_TYPE <MNT1_TYPE>` to "4" for "SToRM32 MavLink" and reboot the autopilot
 - :ref:`SERIAL2_BAUD <SERIAL2_BAUD>` to "115" for 115200 bps.  "SERIAL2" can be replaced with another serial port (i.e. SERIAL1) depending upon the physical connection
 - :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` to 2 for "MAVLink2"
 - :ref:`SR2_EXTRA1 <SR2_EXTRA1>` to 10
 - :ref:`SR2_POSITION <SR2_POSITION>` to 10
 
 The gimbal's maximum angles can be set using these parameters:
```
Comment (Hwurzburg) [inline common/source/docs/common-gremsy-pixyu-gimbal.rst:34]:
```suggestion
The gimbal's maximum angles can be set using these parameters (shown for the first mount):
```

File: common/source/docs/common-gremsy-pixyu-gimbal.rst [hunk 6]
```diff
@@ -25,26 +25,22 @@ We recommend connecting the Gimbals's COM2 port to one of the autopilot's Serial
...
+- :ref:`MNT1_PITCH_MAX <MNT1_PITCH_MAX>` to 30 to allow pointing 30deg up
+- :ref:`MNT1_YAW_MIN <MNT1_YAW_MIN>` to -180 to allow turning around to the left
+- :ref:`MNT1_YAW_MAX <MNT1_YAW_MAX>` to 180 to allow turning around to the right
 
-To control the gimbal's lean angles from a transmitter set:
-
-- :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` to 6 to control the gimbal's tilt (aka pitch angle) with the transmitter's Ch6 tuning knob
-- :ref:`MNT_RC_IN_ROLL <MNT_RC_IN_ROLL>` to some input channel number to control the gimbal's roll angle
-- :ref:`MNT_RC_IN_PAN <MNT_RC_IN_PAN>` to some input channel number to control the gimbals' heading
+To control the gimbal's lean angles from a transmitter set :ref:`RCx_OPTION <RC1_OPTION>` to the respective control axis.
```
Comment (Hwurzburg) [inline common/source/docs/common-gremsy-pixyu-gimbal.rst:43]:
```suggestion
To control the gimbal's lean angles from a transmitter set the RC controls for roll, tilt, or pan using ``RCx_OPTION`` 212 (Mount1 Roll), 213 (Mount1 Pitch), 214 (Mount1 Yaw) for the first mount, or 215-217 for the second mount, respectively.
```

File: common/source/docs/common-gremsy-pixyu-gimbal.rst [hunk 7]
```diff
@@ -100,7 +96,7 @@ Connecting the Gimbals's COM2 port to one of the autopilot's Serial/Telemetry po
 
 Connect with a ground station and set the following parameters:
```
Comment (Hwurzburg) [inline common/source/docs/common-gremsy-pixyu-gimbal.rst:97]:
```suggestion
Connect with a ground station and set the following parameters, if using the first mount::
```

File: common/source/docs/common-simplebgc-gimbal.rst [hunk 8]
```diff
@@ -22,28 +22,28 @@ Although the SimpleBGC can be connected using PWM (similar to the Tarot gimbal)
 
 In ArduPilot/APM Planner/Mission planner set the following variables:
```
Comment (Hwurzburg) [inline common/source/docs/common-simplebgc-gimbal.rst:23]:
```suggestion
In your Ground Control Station set the following variables, if using the first mount::
```

File: common/source/docs/common-simplebgc-gimbal.rst [hunk 9]
```diff
@@ -22,28 +22,28 @@ Although the SimpleBGC can be connected using PWM (similar to the Tarot gimbal)
...
+- :ref:`MNT1_TYPE <MNT1_TYPE>` to 4 / "Mount Type (None, Servo or MavLink)"
+- :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` to 1 / "MavLink" (Note "SERIAL2" should be "SERIAL1" if using Telem1 port, SERIAL4 if using Serial4/5, etc)
 - :ref:`SR2_EXTRA1 <SR2_EXTRA1>` to 20
 - :ref:`SR2_POSITION <SR2_POSITION>` to 10
 - :ref:`SR2_RC_CHAN <SR2_RC_CHAN>` to 20 and all other SR2_* variables to 0.
 
 If you wish to control the pitch angle manually you can set:
 
-- :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` to 6
+- :ref:`RCx_OPTION <RC1_OPTION>` to 213.
```
Comment (Hwurzburg) [inline common/source/docs/common-simplebgc-gimbal.rst:33]:
```suggestion
-  ``RCx_OPTION`` to 213.
```
I dont like using a link to a specific param when discussing a general group....either just unlink and highlight, or put a "if using RC 6..etc."

File: common/source/docs/common-simplebgc-gimbal.rst [hunk 10]
```diff
@@ -22,28 +22,28 @@ Although the SimpleBGC can be connected using PWM (similar to the Tarot gimbal)
...
+    If you connect the gimbal as shown in the above diagram, it is NOT possible to establish a connection to the GUI on a PC via USB and have it connected simultaneously to the autopilot. It happens because the UART1 ("serial" in the picture above) is "paralleled to the onboard USB-UART converter (excepting the "Tiny" boards that has a dedicated USB). If you accidentally connect both the GUI via USB and an autopilot via UART1, you might end up with corrupt data on the flash of the gimbal. It can result in unexpected motor movement! The reason details, please check the `link <https://www.basecamelectronics.com/files/v3/SimpleBGC_32bit_manual_2_6x_eng.pdf>`__.
 
 Setup through the Ground Station
 ================================
 
 Set the following parameters through your ground station and then reboot the autopilot:
```
Comment (Hwurzburg) [inline common/source/docs/common-simplebgc-gimbal.rst:43]:
```suggestion
Set the following parameters through your ground station and then reboot the autopilot( values shown for first mount):
```

File: common/source/docs/common-simplebgc-gimbal.rst [hunk 11]
```diff
@@ -52,22 +52,22 @@ If you are unable to connect you may wish to set the following parameters althou
...
 
-- :ref:`MNT_ANGMIN_ROL <MNT_ANGMIN_ROL>`, :ref:`MNT_ANGMAX_ROL <MNT_ANGMAX_ROL>` to -3000 and 3000 to limit the roll angle to 30 degrees in each direction
-- :ref:`MNT_ANGMIN_TIL <MNT_ANGMIN_TIL>`, :ref:`MNT_ANGMAX_TIL <MNT_ANGMAX_TIL>` to -9000 and 0 to limit the gimbal to point between straight down (-90 degrees) and straight forward (0 degrees)
+- :ref:`MNT1_ROLL_MIN <MNT1_ROLL_MIN>`, :ref:`MNT1_ROLL_MAX <MNT1_ROLL_MAX>` to -30 and 30 to limit the roll angle to 30 degrees in each direction
+- :ref:`MNT1_PITCH_MIN <MNT1_PITCH_MIN>`, :ref:`MNT1_PITCH_MAX <MNT1_PITCH_MAX>` to -90 and 0 to limit the gimbal to point between straight down (-90 degrees) and straight forward (0 degrees)
 
 To control the gimbal's lean angles from a transmitter set:
 
-- :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` to 6 to control the gimbal's tilt (aka pitch angle) with the transmitter's Ch6 tuning knob
+- :ref:`RCx_OPTION <RC1_OPTION>` to 213 to control the gimbal's tilt (aka pitch angle) with the transmitter's Ch6 tuning knob
```
Comment (Hwurzburg) [inline common/source/docs/common-simplebgc-gimbal.rst:60]:
```suggestion
-  set an RC channel's ``RCx_OPTION `` to 213 to control the gimbal's tilt (aka pitch angle)
```

File: common/source/docs/common-storm32-gimbal.rst [hunk 12]
```diff
@@ -51,16 +51,16 @@ Using a ground station (e.g. Mission Planner) set the following parameters.  The
 .. image:: ../../../images/SToRM32_MP_Serial2Baud_new.png
     :target: ../_images/SToRM32_MP_Serial2Baud_new.png
 
-- :ref:`MNT_TYPE <MNT_TYPE>` = 4 (SToRM32 MAVLink) and reboot the autopilot
-- set :ref:`MNT_ANGMIN_PAN <MNT_ANGMIN_PAN>`, :ref:`MNT_ANGMAX_PAN <MNT_ANGMAX_PAN>`, :ref:`MNT_ANGMIN_TIL <MNT_ANGMIN_TIL>`, :ref:`MNT_ANGMAX_TIL <MNT_ANGMAX_TIL>`, :ref:`MNT_ANGMIN_ROL <MNT_ANGMIN_ROL>`, :ref:`MNT_ANGMAX_ROL <MNT_ANGMAX_ROL>` to match your gimbal's range.
-- Optionally set :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` = 6 to control the gimbal's tilt from the transmitter's ch6 tuning knob
+- :ref:`MNT1_TYPE <MNT1_TYPE>` = 4 (SToRM32 MAVLink) and reboot the autopilot
```
Comment (Hwurzburg) [inline common/source/docs/common-storm32-gimbal.rst:54]:
```suggestion
if the first mount is being used, set the following parameters:

- :ref:`MNT1_TYPE <MNT1_TYPE>` = 4 (SToRM32 MAVLink) and reboot the autopilot
```

File: common/source/docs/common-tarot-gimbal.rst [hunk 13]
```diff
@@ -95,11 +95,11 @@ If using the Mission Planner a Camera Gimbal setup screen is available as shown
 
 If manually setting parameters:
```
Comment (Hwurzburg) [inline common/source/docs/common-tarot-gimbal.rst:96]:
```suggestion
If manually setting parameters for the first mount:
```

File: dev/source/docs/mavlink-gimbal-mount.rst [hunk 14]
```diff
@@ -27,8 +27,8 @@ Set the mode with MAV_CMD_DO_MOUNT_CONTROL
 The gimbal/mount's mode can be set by sending a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command and param7 fields set as specified for the `MAV_CMD_DO_MOUNT_CONTROL <https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL>`__ command.
 The supported modes are:
 
-0. Retract : gimbal will move to a retracted position (see MNT_RETRACT_X, Y, Z) or relax
-1. Neutral : gimbal will move to a neutral position (see MNT_NEUTRAL_X, Y, Z)
+0. Retract : gimbal will move to a retracted position (see MNT1_RETRACT_X, Y, Z) or relax
```
Comment (Hwurzburg) [inline dev/source/docs/mavlink-gimbal-mount.rst:30]: @rmackay9 we need to determine if this legacy command only affects Mount1 or not..the new command specifies the target mount....

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-simplebgc-gimbal.rst (hunk 9 above)
Comment (TunaLobster) [reply common/source/docs/common-simplebgc-gimbal.rst:33]: That's the best way I can think of to link to one instance of the parameter list. IMO it is better than not linking to the additional documentation about the parameter.

File: common/source/docs/common-tarot-gimbal.rst [hunk 15]
```diff
@@ -95,11 +95,11 @@ If using the Mission Planner a Camera Gimbal setup screen is available as shown
...
-- :ref:`MNT_TYPE <MNT_TYPE>` = 1 (for "Servo" gimbal)
-- :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` = 6 to specify that RC input channel 6 (normally the tuning knob) will be used to control the pitch/tilt of the gimbal
-- :ref:`MNT_ANGMAX_TIL <MNT_ANGMAX_TIL>` = 0 to specify the gimbal's highest angle is zero degrees (i.e. forward)
-- :ref:`MNT_ANGMIN_TIL <MNT_ANGMIN_TIL>` = -9000 to specify the gimbal's lowest angle is 90 degrees (i.e. down)
-- :ref:`MNT_STAB_TILT <MNT_STAB_TILT>` = 0 to specify ArduPilot should not send tilt compensation to the gimbal (the gimbal does this by itself)
+- :ref:`MNT1_TYPE <MNT1_TYPE>` = 1 (for "Servo" gimbal)
+- :ref:`RC6_OPTION <RC6_OPTION>` = 213 to specify that RC input channel 6 (normally the tuning knob) will be used to control the pitch/tilt of the gimbal
+- :ref:`MNT1_PITCH_MAX <MNT1_PITCH_MAX>` = 0 to specify the gimbal's highest angle is zero degrees (i.e. forward)
+- :ref:`MNT1_PITCH_MIN <MNT1_PITCH_MIN>` = -90 to specify the gimbal's lowest angle is 90 degrees (i.e. down)
+- :ref:`RCx_OPTION <RC1_OPTION>` = 0 to specify ArduPilot should not send tilt compensation to the gimbal (the gimbal does this by itself)
```
Comment (Hwurzburg) [inline common/source/docs/common-tarot-gimbal.rst:102]:
```suggestion
```
there are no longer STAB options

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-tarot-gimbal.rst (hunk 15 above)
Comment (TunaLobster) [reply common/source/docs/common-tarot-gimbal.rst:102]: Fixed

File: common/source/docs/common-simplebgc-gimbal.rst (hunk 9 above)
Comment (Hwurzburg) [reply common/source/docs/common-simplebgc-gimbal.rst:33]: I disagree...it has caused confusion in the past and I have had to answer questions about "do I really need to use that RC channel?"

File: common/source/docs/common-tarot-gimbal.rst (hunk 15 above)
Comment (Hwurzburg) [reply common/source/docs/common-tarot-gimbal.rst:102]: still is there apparently

**Comment by TunaLobster** [comment]
Squashed and credited

File: common/source/docs/common-tarot-gimbal.rst (hunk 15 above)
Comment (TunaLobster) [reply common/source/docs/common-tarot-gimbal.rst]: Had it sitting in my staged. Should be there now.

**Comment by rmackay9** [comment]
Thanks for all this.  4.3 introduces a lot of changes in the gimbal area and it's going to take us a number of PRs to sort it all out.  In any case some of the changes are:

- MNTx_TYPE = 7 (BrushlessPWM) replaces the MNTx_TYPE=1 (Servo) with all the STAB_ params set to zero.
- Let's use "Roll", **"Pitch", "Yaw"** instead of "Roll", **"Tilt", "Pan"**.

**Pull request merged**